apollo-ios-codegen/Sources/GraphQLCompiler/JavaScript/dist/ApolloCodegenFrontend.bundle.js

```diff
@@ -1 +1 @@
)&&void 0!==i?i:[],a=r[o];for(const t of s){const r=t.name.value,i=n[o];xe(i)&&i.getValue(r)?e.reportError(new GraphQLError(`Enum value "${o}.${r}" already exists in the schema. It cannot also be defined in this type extension.`,t.name)):a[r]?e.reportError(new GraphQLError(`Enum value "${o}.${r}" can only be defined once.`,[a[r],t.name])):a[r]=t.name}return!1}},function(e){const t=e.getSchema(),n=t?t.getTypeMap():Object.create(null),r=Object.create(null);return{InputObjectTypeDefinition:i,InputObjectTypeExtension:i,InterfaceTypeDefinition:i,InterfaceTypeExtension:i,ObjectTypeDefinition:i,ObjectTypeExtension:i};function i(t){var i;const o=t.name.value;r[o]||(r[o]=Object.create(null));const s=null!==(i=t.fields)&&void 0!==i?i:[],a=r[o];for(const t of s){const r=t.name.value;jn(n[o],r)?e.reportError(new GraphQLError(`Field "${o}.${r}" already exists in the schema. It cannot also be defined in this type extension.`,t.name)):a[r]?e.reportError(new GraphQLError(`Field "${o}.${r}" can only be defined once.`,[a[r],t.name])):a[r]=t.name}return!1}},function(e){return{DirectiveDefinition(e){var t;const r=null!==(t=e.arguments)&&void 0!==t?t:[];return n(`@${e.name.value}`,r)},InterfaceTypeDefinition:t,InterfaceTypeExtension:t,ObjectTypeDefinition:t,ObjectTypeExtension:t};function t(e){var t;const r=e.name.value,i=null!==(t=e.fields)&&void 0!==t?t:[];for(const e of i){var o;n(`${r}.${e.name.value}`,null!==(o=e.arguments)&&void 0!==o?o:[])}return!1}function n(t,n){const r=$n(n,(e=>e.name.value));for(const[n,i]of r)i.length>1&&e.reportError(new GraphQLError(`Argument "${t}(${n}:)" can only be defined once.`,i.map((e=>e.name))));return!1}},function(e){const t=Object.create(null),n=e.getSchema();return{DirectiveDefinition(r){const i=r.name.value;if(null==n||!n.getDirective(i))return t[i]?e.reportError(new GraphQLError(`There can be only one directive named "@${i}".`,[t[i],r.name])):t[i]=r.name,!1;e.reportError(new GraphQLError(`Directive "@${i}" already exists in the schema. It cannot be redefined.`,r.name))}}},dn,pn,Un,function(e){const t=e.getSchema(),n=Object.create(null);for(const t of e.getDocument().definitions)cn(t)&&(n[t.name.value]=t);return{ScalarTypeExtension:i,ObjectTypeExtension:i,InterfaceTypeExtension:i,UnionTypeExtension:i,EnumTypeExtension:i,InputObjectTypeExtension:i};function i(i){const o=i.name.value,s=n[o],a=null==t?void 0:t.getType(o);let c;if(s?c=Sn[s.kind]:a&&(c=function(e){if(Ae(e))return m.SCALAR_TYPE_EXTENSION;if(De(e))return m.OBJECT_TYPE_EXTENSION;if(we(e))return m.INTERFACE_TYPE_EXTENSION;if(ke(e))return m.UNION_TYPE_EXTENSION;if(xe(e))return m.ENUM_TYPE_EXTENSION;if(Fe(e))return m.INPUT_OBJECT_TYPE_EXTENSION;r(!1,"Unexpected type: "+Y(e))}(a)),c){if(c!==i.kind){const t=function(e){switch(e){case m.SCALAR_TYPE_EXTENSION:return"scalar";case m.OBJECT_TYPE_EXTENSION:return"object";case m.INTERFACE_TYPE_EXTENSION:return"interface";case m.UNION_TYPE_EXTENSION:return"union";case m.ENUM_TYPE_EXTENSION:return"enum";case m.INPUT_OBJECT_TYPE_E
```

Comment:
We should rather be removing this file instead of updating it. The content gets copied into `ApolloCodegenFrontendBundle.swift` anyways.

Reply:
Done